include/translatable-object.php

```diff
@@ -170,7 +170,9 @@ public function set_language( $id, $lang ) {
 
 		$term_taxonomy_ids = wp_set_object_terms( $id, $lang, $this->tax_language );
 
-		wp_cache_set( 'last_changed', microtime(), "{$this->type}s" );
+		if ( $this instanceof PLL_Translated_Object ) {
```

Comment:
Not sure it's a good idea `PLL_Translatable_Object` be aware of its child class `PLL_Translated_Object`. Could the property `cache_type` be declared in `PLL_Translatable_Object`?